apps/address-service/domains/common/utils/healthchecks.js

```diff
@@ -0,0 +1,50 @@
+const { get } = require('lodash')
+
+const conf = require('@open-condo/config')
+
+const { DADATA_PROVIDER } = require('@address-service/domains/common/constants/providers')
+const { DadataSuggestionProvider } = require('@address-service/domains/common/utils/services/suggest/providers/DadataSuggestionProvider')
+
+const getAddressProviderBalanceHealthCheck = () => {
+    return {
+        name: 'address-provider-balance',
+        prepare: () => {
+            const provider = get(conf, 'PROVIDER')
+
+            if (provider === DADATA_PROVIDER) {
+                this.addressProvider = new DadataSuggestionProvider()
+            } else {
+                this.addressProvider = null
+            }
```

Comment:
Maybe it's better to use the [providerDetector](https://github.com/open-condo-software/condo/blob/master/apps/address-service/domains/common/utils/services/providerDetectors.js)?